corehq/form_processor/interfaces/processor.py

```diff
@@ -142,31 +143,35 @@ def xformerror_from_xform_instance(self, instance, error_message, with_new_id=Fa
         return self.processor.xformerror_from_xform_instance(instance, error_message, with_new_id=with_new_id)
 
     def update_responses(self, xform, value_responses_map, user_id):
+        '''
```

Comment:
Not sure why the linter doesn't catch it, but https://www.python.org/dev/peps/pep-0257/ says 

> For consistency, always use """triple double quotes""" around docstrings

Reply:
fixed